llvm/include/llvm/IR/GlobalVariable.h

```diff
@@ -267,6 +267,13 @@ class GlobalVariable : public GlobalObject, public ilist_node<GlobalVariable> {
            getAttributes().hasAttribute("rodata-section");
   }
 
+  MaybeAlign getRequiredGlobalAlignment() {
```

Comment:
This is an llvm CodeGen detail, not something to expose to IR, and may vary based on the exact target and configuration.

Reply:
Done. Moved to `AsmPrinter`. LMK if there's a better place.